src/open_inwoner/ssd/tests/test_views.py

```diff
@@ -52,7 +52,7 @@ def test_get(self):
 
     @patch(
         "open_inwoner.ssd.client.UitkeringClient.get_report",
-        return_value=mock_report(str(FILES_DIR / "uitkering_response.xml")),
+        return_value=mock_report(str(FILES_DIR / "uitkering1.xml")),
```

Comment:
What is the difference between the different uitkering files? Lets give these files a name with something more readable then just a number. 

If the language supports it (XML does) you could also add a comment on top inside the file and at the location of any interesting changes.